.eslintrc

```diff
@@ -13,7 +13,29 @@
     "react"
   ],
   "ecmaFeatures": {
-    "jsx": true
+    "jsx": true,
+    "arrowFunctions": true,
+    "binaryLiterals": true,
+    "blockBindings": true,
+    "classes": true,
+    "defaultParams": true,
+    "destructuring": true,
+    "forOf": true,
+    "generators": true,
```

Comment:
@hxoliverrumbelow did you mention in the SA meeting that you would not want to allow all es6 features, I think `generators` was one you weren't keen on?